project/assets/main/locale/es.po

```diff
@@ -18,2065 +18,2086 @@ msgstr ""
 msgstr ""
 
-#: localizables-extracted.py:78
+#: project/assets/main/locale/localizables-extracted.py:79
 msgid "Well... I wouldn't eat it."
 msgstr ""
 
-#: localizables-extracted.py:79
-msgid "Wow,/ I think you killed it!/ You have some real talent."
+#: project/assets/main/locale/localizables-extracted.py:80
+msgid "Wow, I think you killed it! You have some real talent."
 msgstr ""
 
-#: localizables-extracted.py:80
+#: project/assets/main/locale/localizables-extracted.py:81
 msgid "Whoa, thanks!"
 msgstr ""
 
-#: localizables-extracted.py:81
+#: project/assets/main/locale/localizables-extracted.py:82
 msgid ""
-"Hey,/ that was pretty good!/ And y'know,/ with a little more training you'll "
+"Hey, that was pretty good! And y'know, with a little more training you'll "
 "get even better."
 msgstr ""
 
-#: localizables-extracted.py:82
+#: project/assets/main/locale/localizables-extracted.py:83
 msgid "Oh okay cool!"
 msgstr ""
 
-#: localizables-extracted.py:83 localizables-extracted.py:210
-msgid "./././"
-msgstr ""
-
-#: localizables-extracted.py:84
-msgid "./././?"
+#: project/assets/main/locale/localizables-extracted.py:85
+msgid "...?"
 msgstr ""
 
-#: localizables-extracted.py:85
-msgid "./././ ./././"
+#: project/assets/main/locale/localizables-extracted.py:86
+msgid "... ..."
 msgstr ""
 
-#: localizables-extracted.py:86
-msgid "./././ ./././ ./././"
+#: project/assets/main/locale/localizables-extracted.py:87
+msgid "... ... ..."
 msgstr ""
 
-#: localizables-extracted.py:87
+#: project/assets/main/locale/localizables-extracted.py:88
 msgid ""
-"Well./././ I wouldn't eat it. Did you have any cooking experience before "
-"this?"
+"Well... I wouldn't eat it. Did you have any cooking experience before this?"
 msgstr ""
 
-#: localizables-extracted.py:88
+#: project/assets/main/locale/localizables-extracted.py:89
 msgid ""
-"Uhh I mean like,/ Richie's been helping me figure out the kitchen "
-"stuff./ ./././Did I mess something up?"
+"Uhh I mean like, Richie's been helping me figure out the kitchen stuff. ..."
+"Did I mess something up?"
 msgstr ""
 
-#: localizables-extracted.py:89
-msgid "Hmm./ Yes!/ Pretty good./ Your rhythm's ahhh./././ a little off though!"
+#: project/assets/main/locale/localizables-extracted.py:90
+msgid "Hmm. Yes! Pretty good. Your rhythm's ahhh... a little off though!"
 msgstr ""
 
-#: localizables-extracted.py:90 localizables-extracted.py:94
-msgid "My./././ my rhythm?"
+#: project/assets/main/locale/localizables-extracted.py:91
+#: project/assets/main/locale/localizables-extracted.py:95
+msgid "My... my rhythm?"
 msgstr ""
 
-#: localizables-extracted.py:91 localizables-extracted.py:95
+#: project/assets/main/locale/localizables-extracted.py:92
+#: project/assets/main/locale/localizables-extracted.py:96
 msgid ""
-"Yes! That food should be coming out like pap-/pap-/pap-/pap-/pap!/ Right now "
-"you're more like pappap,// pappa/ pap."
+"Yes! That food should be coming out like pap-pap-pap-pap-pap! Right now "
```

Comment:
Do you inject pauses between hyphenated words?

I'm OK with this sentence changing its cadence, but more concerned with hyphens pausing.

Reply:
Yes, I was injecting pauses between hyphenated words.

I've corrected the relevant code to avoid inserting pauses in hyphenated words. I've also reverted this line of dialog to preserve the pauses; the pauses seem important.